<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(26, 36, 33);</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title_label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>551</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 36pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Hamamatsu UV Control</string>
    </property>
   </widget>
   <widget class="QPushButton" name="onOff_button">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>100</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(19, 38, 77);
	color: rgb(183, 225, 161);
   }

 QPushButton::hover {
	background-color: rgb(181, 234, 170);
	color: rgb(0, 0, 0);
   }
</string>
    </property>
    <property name="text">
     <string>On/OFF</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="onOff_indicator">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>80</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">selection-color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="intensity_control">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>230</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSpinBox{
	background-color: rgb(19, 38, 77);
	border-color: rgb(181, 234, 170);
	border-width: 1px;
    border-style: solid;
	
	color: rgb(255, 255, 255);
	padding: 1px 0px 1px 3px;
}
QSpinBox QAbstractItemView {
	color: rgb(255, 255, 255);
            }</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="timer_label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 700 10pt &quot;Segoe UI&quot;;
text-decoration: underline;</string>
    </property>
    <property name="text">
     <string>Timer</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="timer_input">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-color: rgb(19, 38, 77);
	border-color: rgb(181, 234, 170);
	border-width: 1px;
    border-style: solid;
	
	color: rgb(255, 255, 255);
	padding: 1px 0px 1px 3px;
}
QLineEdit QAbstractItemView {
	color: rgb(255, 255, 255);
            }</string>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confirmTimer_button">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>79</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(19, 38, 77);
	color: rgb(183, 225, 161);
   }

 QPushButton::hover {
	background-color: rgb(181, 234, 170);
	color: rgb(0, 0, 0);
   }
</string>
    </property>
    <property name="text">
     <string>Start Timer</string>
    </property>
   </widget>
   <widget class="QLabel" name="timer_display">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>160</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);
font: 20pt &quot;OCR A&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="manual_box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Manual Mode</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 700 10pt &quot;Segoe UI&quot;;
text-decoration: underline;</string>
    </property>
    <property name="text">
     <string>Channel Selection</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>200</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 700 10pt &quot;Segoe UI&quot;;
text-decoration: underline;</string>
    </property>
    <property name="text">
     <string>Current Intensity</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>101</width>
      <height>116</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="1">
      <widget class="QCheckBox" name="ch1_indicator">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">selection-color: rgb(0, 85, 255);
selection-background-color: rgb(0, 170, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="ch2_button">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(19, 38, 77);
	color: rgb(183, 225, 161);
   }
QPushButton:checked {
	background-color: rgb(181, 234, 170);
    color: rgb(0, 0, 0);
}
 QPushButton::hover {
	background-color: rgb(181, 234, 170);
	color: rgb(0, 0, 0);
   }
</string>
       </property>
       <property name="text">
        <string>CH2</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="ch2_indicator">
       <property name="styleSheet">
        <string notr="true">selection-color: rgb(0, 85, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="ch3_button">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(19, 38, 77);
	color: rgb(183, 225, 161);
   }
QPushButton:checked {
	background-color: rgb(181, 234, 170);
    color: rgb(0, 0, 0);
}
 QPushButton::hover {
	background-color: rgb(181, 234, 170);
	color: rgb(0, 0, 0);
   }
</string>
       </property>
       <property name="text">
        <string>CH3</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="ch3_indicator">
       <property name="styleSheet">
        <string notr="true">selection-color: rgb(0, 85, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="ch4_button">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(19, 38, 77);
	color: rgb(183, 225, 161);
   }
QPushButton:checked {
	background-color: rgb(181, 234, 170);
    color: rgb(0, 0, 0);
}
 QPushButton::hover {
	background-color: rgb(181, 234, 170);
	color: rgb(0, 0, 0);
   }
</string>
       </property>
       <property name="text">
        <string>CH4</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="ch4_indicator">
       <property name="styleSheet">
        <string notr="true">selection-color: rgb(0, 85, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="ch1_button">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(19, 38, 77);
	color: rgb(183, 225, 161);
   }
QPushButton:checked {
	background-color: rgb(181, 234, 170);
    color: rgb(0, 0, 0);
}
 QPushButton::hover {
	background-color: rgb(181, 234, 170);
	color: rgb(0, 0, 0);
   }

</string>
       </property>
       <property name="text">
        <string>CH1</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>380</y>
      <width>481</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="2">
      <widget class="QSpinBox" name="step3_intensity">
       <property name="styleSheet">
        <string notr="true">QSpinBox{
	background-color: rgb(19, 38, 77);
	border-color: rgb(181, 234, 170);
	border-width: 1px;
    border-style: solid;
	
	color: rgb(255, 255, 255);
	padding: 1px 0px 1px 3px;
}
QSpinBox QAbstractItemView {
	color: rgb(255, 255, 255);
            }</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="step4_time">
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	background-color: rgb(19, 38, 77);
	border-color: rgb(181, 234, 170);
	border-width: 1px;
    border-style: solid;
	
	color: rgb(255, 255, 255);
	padding: 1px 0px 1px 3px;
}
QLineEdit QAbstractItemView {
	color: rgb(255, 255, 255);
            }</string>
       </property>
       <property name="text">
        <string>00:00:00</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="step3_time">
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	background-color: rgb(19, 38, 77);
	border-color: rgb(181, 234, 170);
	border-width: 1px;
    border-style: solid;
	
	color: rgb(255, 255, 255);
	padding: 1px 0px 1px 3px;
}
QLineEdit QAbstractItemView {
	color: rgb(255, 255, 255);
            }</string>
       </property>
       <property name="text">
        <string>00:00:00</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="step1_time">
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	background-color: rgb(19, 38, 77);
	border-color: rgb(181, 234, 170);
	border-width: 1px;
    border-style: solid;
	
	color: rgb(255, 255, 255);
	padding: 1px 0px 1px 3px;
}
QLineEdit QAbstractItemView {
	color: rgb(255, 255, 255);
            }</string>
       </property>
       <property name="text">
        <string>00:00:00</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QSpinBox" name="step4_intensity">
       <property name="styleSheet">
        <string notr="true">QSpinBox{
	background-color: rgb(19, 38, 77);
	border-color: rgb(181, 234, 170);
	border-width: 1px;
    border-style: solid;
	
	color: rgb(255, 255, 255);
	padding: 1px 0px 1px 3px;
}
QSpinBox QAbstractItemView {
	color: rgb(255, 255, 255);
            }</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="step_control_button">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(19, 38, 77);
	color: rgb(183, 225, 161);
   }

 QPushButton::hover {
	background-color: rgb(181, 234, 170);
	color: rgb(0, 0, 0);
   }
</string>
       </property>
       <property name="text">
        <string>Confirm</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="step2_intensity">
       <property name="styleSheet">
        <string notr="true">QSpinBox{
	background-color: rgb(19, 38, 77);
	border-color: rgb(181, 234, 170);
	border-width: 1px;
    border-style: solid;
	
	color: rgb(255, 255, 255);
	padding: 1px 0px 1px 3px;
}
QSpinBox QAbstractItemView {
	color: rgb(255, 255, 255);
            }</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLineEdit" name="step5_time">
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	background-color: rgb(19, 38, 77);
	border-color: rgb(181, 234, 170);
	border-width: 1px;
    border-style: solid;
	
	color: rgb(255, 255, 255);
	padding: 1px 0px 1px 3px;
}
QLineEdit QAbstractItemView {
	color: rgb(255, 255, 255);
            }</string>
       </property>
       <property name="text">
        <string>00:00:00</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QSpinBox" name="step5_intensity">
       <property name="styleSheet">
        <string notr="true">QSpinBox{
	background-color: rgb(19, 38, 77);
	border-color: rgb(181, 234, 170);
	border-width: 1px;
    border-style: solid;
	
	color: rgb(255, 255, 255);
	padding: 1px 0px 1px 3px;
}
QSpinBox QAbstractItemView {
	color: rgb(255, 255, 255);
            }</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QSpinBox" name="step1_intensity">
       <property name="styleSheet">
        <string notr="true">QSpinBox{
	background-color: rgb(19, 38, 77);
	border-color: rgb(181, 234, 170);
	border-width: 1px;
    border-style: solid;
	
	color: rgb(255, 255, 255);
	padding: 1px 0px 1px 3px;
}
QSpinBox QAbstractItemView {
	color: rgb(255, 255, 255);
            }</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="step2_time">
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	background-color: rgb(19, 38, 77);
	border-color: rgb(181, 234, 170);
	border-width: 1px;
    border-style: solid;
	
	color: rgb(255, 255, 255);
	padding: 1px 0px 1px 3px;
}
QLineEdit QAbstractItemView {
	color: rgb(255, 255, 255);
            }</string>
       </property>
       <property name="text">
        <string>00:00:00</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="step_display">
       <property name="styleSheet">
        <string notr="true">QLabel{
	background-color: rgb(19, 38, 77);
	border-color: rgb(181, 234, 170);
	border-width: 1px;
    border-style: solid;
	
	color: rgb(255, 255, 255);
	padding: 1px 0px 1px 3px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="step_indicator">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>360</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>160</width>
      <height>116</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_3">
     <item row="0" column="0">
      <widget class="QPushButton" name="saveIntensity1_button">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(19, 38, 77);
	color: rgb(183, 225, 161);
   }

 QPushButton::hover {
	background-color: rgb(181, 234, 170);
	color: rgb(0, 0, 0);
   }
</string>
       </property>
       <property name="text">
        <string>Save </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="setIntensity1_button">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(19, 38, 77);
	color: rgb(183, 225, 161);
   }

 QPushButton::hover {
	background-color: rgb(181, 234, 170);
	color: rgb(0, 0, 0);
   }
</string>
       </property>
       <property name="text">
        <string>0%</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="saveIntensity2_button">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(19, 38, 77);
	color: rgb(183, 225, 161);
   }

 QPushButton::hover {
	background-color: rgb(181, 234, 170);
	color: rgb(0, 0, 0);
   }
</string>
       </property>
       <property name="text">
        <string>Save </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="setIntensity2_button">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(19, 38, 77);
	color: rgb(183, 225, 161);
   }

 QPushButton::hover {
	background-color: rgb(181, 234, 170);
	color: rgb(0, 0, 0);
   }
</string>
       </property>
       <property name="text">
        <string>10%</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="saveIntensity3_button">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(19, 38, 77);
	color: rgb(183, 225, 161);
   }

 QPushButton::hover {
	background-color: rgb(181, 234, 170);
	color: rgb(0, 0, 0);
   }
</string>
       </property>
       <property name="text">
        <string>Save </string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="setIntensity3_button">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(19, 38, 77);
	color: rgb(183, 225, 161);
   }

 QPushButton::hover {
	background-color: rgb(181, 234, 170);
	color: rgb(0, 0, 0);
   }
</string>
       </property>
       <property name="text">
        <string>50%</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="saveIntensity4_button">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(19, 38, 77);
	color: rgb(183, 225, 161);
   }

 QPushButton::hover {
	background-color: rgb(181, 234, 170);
	color: rgb(0, 0, 0);
   }
</string>
       </property>
       <property name="text">
        <string>Save </string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="setIntensity4_button">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(19, 38, 77);
	color: rgb(183, 225, 161);
   }

 QPushButton::hover {
	background-color: rgb(181, 234, 170);
	color: rgb(0, 0, 0);
   }
</string>
       </property>
       <property name="text">
        <string>100%</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="time_comboBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>0</number>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
	background-color: rgb(19, 38, 77);
	border-color: rgb(181, 234, 170);
	border-width: 1px;
    border-style: solid;
	
	color: rgb(255, 255, 255);
	padding: 1px 0px 1px 3px;
}
QComboBox QAbstractItemView {
	color: rgb(255, 255, 255);
            }</string>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Time Selection</string>
    </property>
    <item>
     <property name="text">
      <string>3 minutes</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5 minutes</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20 minutes</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="step_comboBox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>340</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
	background-color: rgb(19, 38, 77);
	border-color: rgb(181, 234, 170);
	border-width: 1px;
    border-style: solid;
	
	color: rgb(255, 255, 255);
	padding: 1px 0px 1px 3px;
}
QComboBox QAbstractItemView {
	color: rgb(255, 255, 255);
            }</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Procedure Selection</string>
    </property>
    <item>
     <property name="text">
      <string>+ Add New Procedure</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="save_procedure_button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>310</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(19, 38, 77);
	color: rgb(183, 225, 161);
   }

 QPushButton::hover {
	background-color: rgb(181, 234, 170);
	color: rgb(0, 0, 0);
   }
</string>
    </property>
    <property name="text">
     <string>Save Procedure</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="save_procedure_input">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>340</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	background-color: rgb(19, 38, 77);
	border-color: rgb(181, 234, 170);
	border-width: 1px;
    border-style: solid;
	
	color: rgb(255, 255, 255);
	padding: 1px 0px 1px 3px;
}
QLineEdit QAbstractItemView {
	color: rgb(255, 255, 255);
            }</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Please Input Name</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>